### ISSUE TITLE
[Config Support]: go2rtc config

### ISSUE THREAD

**Issue by primski** [blakeblackshear/frigate#9646]
### Describe the problem you are having

Go2RTC is not working

### Version

stable

### Frigate config file

```
logger:
  default: info
     
mqtt:
  host: 10.0.23.11
  port: 15883

objects:
  track:
    - person
    - car
    - motorcycle
    - bicycle
    - dog
    - cat
    - bird
    - horse
    - sheep
    - cow
    - bear

birdseye:
  enabled: True
  width: 1280
  height: 720
  quality: 8
  mode: continuous

cameras:
  front:
    ffmpeg:
      output_args:
        record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
      inputs:
        - path: rtsp://10.0.23.11:19554/front
          roles:
            - record
        - path: rtsp://10.0.23.11:19554/front_sub
          roles:
           - detect

    rtmp:
      enabled: False
    detect:
      width: 800
      height: 448
      fps: 12

    record:
      enabled: True
      retain:
        days: 30
        mode: motion
     
    motion:
      mask:
        - 51,65,297,21,460,53,461,0,274,0,49,0    
    snapshots:
      enabled: True
      retain:
        default: 30
    
ffmpeg:
  hwaccel_args: -hwaccel vaapi -hwaccel_device /dev/dri/renderD128 -hwaccel_output_format yuv420p

go2rtc:
  streams:
    front:
      rtsp://aaa:bbb@10.0.23.50:554/user=admin_password=aaa_bbb=1_stream=0&protocol=unicast.sdp?real_stream
    front_sub:
      rtsp://aaa:bbb@10.0.23.50:554/user=admin_password=aaa_bbb=1_stream=1&protocol=unicast.sdp?real_stream
```
### Relevant log output

I only this in go2rtc log:
```
2024-02-04 10:52:16.185764335  [INFO] Preparing new go2rtc config...
2024-02-04 10:52:16.458792992  [INFO] Starting go2rtc...
2024-02-04 10:52:16.555475986  10:52:16.555 INF go2rtc version 1.8.4 linux/amd64
2024-02-04 10:52:16.555907374  10:52:16.555 INF [rtsp] listen addr=:8554
2024-02-04 10:52:16.555932504  10:52:16.555 INF [api] listen addr=:1984
2024-02-04 10:52:16.556060192  10:52:16.556 INF [webrtc] listen addr=:8555
2024-02-04 10:52:26.186896115  [INFO] Starting go2rtc healthcheck service...
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

CPU (no coral)

### Any other information that may be helpful

I am running inside docker, here is the compose file, all ports open, i think...

```
version: '3.9'

services:
  frigate:
    # image: blakeblackshear/frigate:0.11.1
    image: ghcr.io/blakeblackshear/frigate:stable
    container_name: frigate-security
    volumes:
      - ./data/frigate/config:/config
      - ./data/frigate/media:/media/frigate
      - type: tmpfs
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    devices:
      - /dev/dri/renderD128
    ports:
      - 11935:1935
      - 15000:5000
      - 15001:5001
      - 18080:8080
      - 19984:1984
      - 19554:8554
      - 19555:8555
    networks:
      - proxy-tier
    dns:
      - 192.168.128.1
    restart: always
    shm_size: '128m'

networks:
  proxy-tier:
    name: internal_network
    external: true

```

MQTT:
```
version: '3.9'

services:
  mqtt:
    container_name: mqtt
    image: eclipse-mosquitto:1.6
    restart: unless-stopped
    ports:
      - 15901:9001
      - 15883:1883
    networks:
      - proxy-tier

networks:
  proxy-tier:
    name: internal_network
    external: true

```

**Comment by NickM-27** [comment]
in what way is it not working?

**Comment by NickM-27** [comment]
extra configuration is required for webrtc, please read the docs https://docs.frigate.video/configuration/live#webrtc-extra-configuration

**Comment by primski** [comment]
Thank you sir, I tried it but cannot get it to work ... it also says there this will not work behind nginx reverse proxy and my frigate is behind nginx so it might be that ...

not that important though, frigate works now on the web, in my Home Assistant and I also integrated the notifications blueprint and that works too. 

Some browsers have problems playing because the camera is in x265, i figured webrtc might fix this? anyways, thanks!
